the serviceMonitor is automatically created by the operator

lzecca78 opened the pull request:
I discovered that the Jaeger Operator creates (silently) already a ServiceMonitor that use the same labelSelector of the one I added manually:

https://github.com/jaegertracing/jaeger-operator/blob/050e95a671d5cfceda848055d61af0968f5f68a3/pkg/cmd/start/bootstrap.go#L361-L378

For this reason the manually added serviceMonitor ends up in creating duplication of metrics and triggering alerts.
Removing it.

ralgozino reviewed (approved):
the change lgtm, I don't remember if we made official using the `docs/releases/unreleased.md` file to track the changes introduced by the PRs, can you confirm @angelbarrera92 ? that would be the only thing missing in that case

angelbarrera92 commented:
> the change lgtm, I don't remember if we made official using the `docs/releases/unreleased.md` file to track the changes introduced by the PRs, can you confirm @angelbarrera92 ? that would be the only thing missing in that case

Ye, we are using it, nothing wrote in stone, but ye... we should use it

nutellinoit reviewed (approved):
LGTM! :tada:

lnovara reviewed (approved):
LGTM :shipit:

lzecca78 commented:
hi @ralgozino! I tried to solved the unreleased issues, introducing the git-chglog. I guess that this is a more wide improvement that can be easily spread in all our repo, but for sure this is a good starting point! let me know what do you think so we can go ahead and merge this PR

nutellinoit commented:
Hi @lzecca78 , can we remove the git-chglog from this PR?
If we want to use this approach we need to make it standard for all modules.

lzecca78 commented:
Yes @nutellinoit , definitely!

ralgozino reviewed (changes requested):
as said by @nutellinoit

lnovara commented:
@lzecca78 do you need any help with this PR?

lzecca78 commented:
@lnovara mmmm I totally lost the context of this PR (is almost 1 year ago and in the meanwhile, a lot of things happened). So definitely would be better do this stuff with someone else that has a more recent experience with this module.

lzecca78 commented:
@nutellinoit , @ralgozino , @lnovara i removed the git-chglog and fixed conflicts with master branch merging master in the branch. Now it should be ready  :)! Let me know if everything is fine now

nutellinoit commented on CHANGELOG.md:
```diff
@@ -0,0 +1,56 @@
+<a name="unreleased"></a>
```
This file should be removed

nutellinoit commented on README.md:
```diff
@@ -71,16 +68,34 @@ following content:
 
 For further details please refer to the single package directories in this repository.
 
+## CHANGELOG
```
Also these instruction on the main readme should be removed

nutellinoit commented on katalog/istio-operator/jaeger-operator/kustomization.yaml:
```diff
@@ -7,9 +7,13 @@ resources:
   - cluster_role.yaml
   - cluster_role_binding.yaml
   - ns.yml
+  - https://raw.githubusercontent.com/jaegertracing/jaeger-operator/v1.22.1/deploy/crds/jaegertracing.io_jaegers_crd.yaml
```
Are these files duplicated? By the way we merged #13 that removes remote manifests dependencies, i think we should remove all these remote links

nutellinoit reviewed (changes requested):
Left some comments.

lnovara reviewed (approved):
LGTM! :shipit:

lnovara commented:
@ralgozino @nutellinoit I think we can merge this! 🚀

ralgozino reviewed (approved):
thank you @lzecca78 !
